apps/dokploy/templates/appwrite/docker-compose.yml

```diff
@@ -0,0 +1,801 @@
+# Usage
+#   CHECK all the environment variables that you'll need.
+#   Replace YOUR_DOMAIN_HERE and YOUR_FUNCTION_DOMAIN_HERE in this docker compose file and in the environmente variables
+#   If you need Clamav, uncomment the lines required below
+#   Note: Updates needs to be tested/adjusted properly - See Appwrite docs about updating (https://appwrite.io/docs/advanced/self-hosting/update)
+
+x-logging: &a1
+  logging:
+    driver: json-file
+    options:
+      max-file: "5"
+      max-size: 10m
+services:
+  appwrite:
+    image: appwrite/appwrite:1.6.0
+    <<: *a1
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+    labels:
+      - traefik.enable=true
+      - traefik.http.services.appwrite_api.loadbalancer.server.port=80
+    
+      - traefik.http.routers.awmain.rule=Host(`YOUR_DOMAIN_HERE`) && !PathPrefix(`/console`) && !PathPrefix(`/v1/realtime`)
+      - traefik.http.routers.awmain.entrypoints=web,websecure
+      - traefik.http.routers.awmain.tls=true
+      - traefik.http.routers.awmain.tls.certresolver=letsencrypt
+      - traefik.http.routers.awmain.service=appwrite_api
+      
+      - traefik.http.routers.awserverf.rule=Host(`YOUR_FUNCTION_DOMAIN_HERE`)
+      - traefik.http.routers.awserverf.entrypoints=web,websecure
+      - traefik.http.routers.awserverf.tls=true
+      - traefik.http.routers.awserverf.tls.certresolver=letsencrypt
+      - traefik.http.routers.awserverf.service=appwrite_api
```

Comment:
Why this manual labels?

Reply:
In appwrite it seems you need to redirect traefik to the appwrite container for all route but not for console  or realtime (which goes to their respective containers), otherwise you'll face some issues with the admin area and wirh some websocket connections (that rely on the realtime endpoint)

When I made the template I faced the issue that the domain tab didn't worked (at least for me it didn't) so I had to put the manual traefik labels to make it work.

About the function_domain labels: in some cases the function domain can be different from the regular API domains, so you need to explicitly add it in the traefik routers.